requirements-dev.txt

```diff
@@ -1,4 +1,4 @@
-docker-py>=0.3.2
+docker-py>=1.7.0
```

Comment:
1.7.0 isn't out yet - if you change this to `docker-py==1.7.0rc2` it should work. That seems to be why CI is currently failing.